File: tests/core/core_libraryset/main_test.cs
using System;
using System.IO;
using System.Reflection;

using Xunit;

namespace example_test
{
    public class MyTest
    {
        [Fact]
        public void MyTest2()
        {
            var src = Path.Combine(Path.GetDirectoryName(typeof(MyTest).Assembly.Location), "test.txt");
            Assert.True(File.Exists(src));
        }
    }
}

File: tests/core/core_libraryset/main_test.fs
module example_test

open System
open System.IO
open System.Reflection

open Xunit

[<Fact>]
let ``MyTest2`` () =
    let src =
        Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "test.txt")

    Assert.True(File.Exists(src))